dcos_installer/upgrade.py

```diff
@@ -111,8 +119,13 @@
 fi
 
 echo "Upgrading DC/OS $role_name {{ installed_cluster_version }} -> {{ installer_version }}"
-pkgpanda fetch --repository-url={{ bootstrap_url }} {{ cluster_packages }} > /dev/null
-pkgpanda activate --no-block {{ cluster_packages }} > /dev/null
+if [[ "$VERBOSE" = "true" ]]; then
+    pkgpanda fetch --repository-url={{ bootstrap_url }} {{ cluster_packages }}
+    pkgpanda activate --no-block {{ cluster_packages }}
+else
+    pkgpanda fetch --repository-url={{ bootstrap_url }} {{ cluster_packages }} > /dev/null
+    pkgpanda activate --no-block {{ cluster_packages }} > /dev/null
+fi
```

Comment:
Duplicating the Pkgpanda commands here feels a little hacky, and it's going to blow up the length of this script -- that `cluster_packages` var is pretty long. We could do this conditional redirect by sending output to a new file descriptor, and conditionally attaching that file descriptor to either stdout or `/dev/null`. Something like this:

```
<snip>

if [[ "$VERBOSE" = "true" ]]; then
    exec 3>&1
else
    exec 3>/dev/null
fi

<snip>

pkgpanda fetch --repository-url={{ bootstrap_url }} {{ cluster_packages }} >&3
pkgpanda activate --no-block {{ cluster_packages }} >&3

<snip>
```